### DISCUSSION TITLE
RX-V681 works

### DISCUSSION THREAD

**Discussion by rajchelm** [mvdwetering/yamaha_ynca#254] [General]
RX-V681 works fine :)
![2024-03-08_12-29](https://github.com/mvdwetering/yamaha_ynca/assets/49623773/31441a87-f33c-41b5-b868-e90bafa81392)

I didn't find an option to switch saved radio stations - is it possible to add such an option as in the device's UI?
![2024-03-08_12-37](https://github.com/mvdwetering/yamaha_ynca/assets/49623773/151b7129-55d3-4951-835d-dacb1a6f79a6)

When I change the radio preset to 12 the request is sent:
to: `/YamahaRemoteControl/ctrl`
payload: `<YAMAHA_AV cmd="PUT"><Tuner><Play_Control><Preset><Preset_Sel>12</Preset_Sel></Preset></Play_Control></Tuner></YAMAHA_AV>`

**Comment by mvdwetering** [comment]
Thanks for the update, I added RX-V681 to the list in the readme.

The command you posted is a different protocol from what this integration is using. But you are in luck, I have been working on adding support for presets recently. It is not done yet, still need to wrap up some testing, but will be in next release.
